src/main/java/com/stripe/model/Account.java

```diff
@@ -21,7 +21,7 @@
 @Setter
 @EqualsAndHashCode(callSuper = false)
 public class Account extends APIResource implements HasId, MetadataStore<Account> {
-  String id;
+  @Getter(onMethod = @__({@Override})) String id;
```

Comment:
There is a slightly better syntax for this: `@Getter(onMethod_ = {@Override})`, but it requires Java 8+ :/

Reply:
Ah, too bad :/